src/plugins/discover/public/application/main/components/top_nav/get_top_nav_links.tsx

```diff
@@ -103,7 +103,8 @@ export const getTopNavLinks = ({
     testId: 'discoverSaveButton',
     iconType: 'save',
     emphasize: true,
-    run: () => onSaveSearch({ savedSearch, services, indexPattern, navigateTo, state }),
+    run: (anchorElement: HTMLElement) =>
+      onSaveSearch({ savedSearch, services, indexPattern, navigateTo, state, anchorElement }),
```

Comment:
Thanks for the suggestion, I'll look into this

Reply:
I originally passed `anchorElement` to `onSaveSearch` to match the popover functions, but I think since this one is a modal it doesn't make sense to have an `anchorElement` anyway, so I added your suggestion. Thanks!